src/commands/flinkUDFs.ts

```diff
@@ -15,5 +25,86 @@ export function registerFlinkUDFCommands(): Disposable[] {
       "confluent.flinkdatabase.setUDFsViewMode",
       setFlinkUDFViewModeCommand,
     ),
+    registerCommandWithLogging(
+      "confluent.artifacts.createUdfRegistrationDocument",
+      createUdfRegistrationDocumentCommand,
+    ),
+    registerCommandWithLogging(
+      "confluent.artifacts.startGuidedUdfCreation",
+      startGuidedUdfCreationCommand,
+    ),
   ];
 }
+export async function createUdfRegistrationDocumentCommand(selectedArtifact: FlinkArtifact) {
```

Comment:
Minor: I'd put these above `registerFlinkUDFCommands()` or move everything below it -- currently `registerFlinkUDFCommands()` is wedged in the middle